components/eamxx/src/share/grid/remap/coarsening_remapper.cpp

```diff
@@ -3,6 +3,7 @@
 #include "share/grid/point_grid.hpp"
 #include "share/grid/grid_import_export.hpp"
 #include "share/io/scorpio_input.hpp"
+#include "share/field/field.hpp"
```

Comment:
Oddly enough changing `m_is_read_only` to `mutable` affected the linking of this file.